libraries/AP_RangeFinder/AP_RangeFinder_Benewake_TFS20L.cpp

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (C) 2024
+ *
+ * This file is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This file is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "AP_RangeFinder_Benewake_TFS20L.h"
+
+#if AP_RANGEFINDER_BENEWAKE_TFS20L_ENABLED
+
+#include <utility>
+
+#include <GCS_MAVLink/GCS.h>
+#include <AP_HAL/AP_HAL.h>
+
+extern const AP_HAL::HAL& hal;
+
+#define DRIVER "TFS20L"
+#define BENEWAKE_OUT_OF_RANGE_ADD_CM 100
+
+
+AP_RangeFinder_Benewake_TFS20L::AP_RangeFinder_Benewake_TFS20L(
+        RangeFinder::RangeFinder_State &_state,
+        AP_RangeFinder_Params &_params,
+        AP_HAL::I2CDevice *dev)
+    : AP_RangeFinder_Backend(_state, _params)
+    , _dev(dev)
+{
+}
+
+AP_RangeFinder_Backend *AP_RangeFinder_Benewake_TFS20L::detect(
+        RangeFinder::RangeFinder_State &_state,
+        AP_RangeFinder_Params &_params,
+        AP_HAL::I2CDevice *dev)
+{
+    if (!dev) return nullptr;
```

Comment:
```suggestion
    if (dev == nullptr) {
        return nullptr;
    }
```

see our style guide.